CRM/Case/BAO/Case.php

```diff
@@ -1041,10 +1041,11 @@ public static function getCaseActivity($caseID, &$params, $contactID, $context =
 
     // define statuses which are handled like Completed status (others are assumed to be handled like Scheduled status)
     $compStatusValues = array();
-    $compStatusNames = array('Completed', 'Left Message', 'Cancelled', 'Unreachable', 'Not Required');
+    $compStatusNames = array('Left Message', 'Cancelled', 'Unreachable', 'Not Required');
```

Comment:
I'm still not happy with this made-up list. IMO we should be using 

    CRM_Activity_BAO_Activity::getStatusesByType(CRM_Activity_BAO_Activity::COMPLETED) + CRM_Activity_BAO_Activity::getStatusesByType(CRM_Activity_BAO_Activity::CANCELLED)

Reply:
Basically whoever made up that list made some arbitrary decisions about some ambiguous statuses. Is "Left Message" considered completed, or not? Well it depends on the use-case really. Which is why we should allow it to be user-configurable and change the code to respect the configured options rather than a hard-coded list IMO.

Reply:
Updated the PR with the above changes. Only `$compStatusValues` is used now which hold the ids of `COMPLETED` and `CANCELLED` activity types 👍